project/project.go

```diff
@@ -219,8 +233,14 @@ func (v Project) CleanPublishedCache() error {
 }
 
 // PublishedReference forms published reference for specific branch.
-func (v Project) PublishedReference(branch string) string {
-	pub := config.RefsPub + branch
+func (v Project) PublishedReference(branch *ReviewableBranch) string {
+	var (
+		branchName        = branch.Branch.Name
+		remoteTrackBranch = branch.RemoteTrack.Branch
+	)
+	branchName = strings.TrimPrefix(branchName, config.RefsHeads)
+	remoteTrackBranch = strings.TrimPrefix(remoteTrackBranch, config.RefsHeads)
+	pub := config.RefsPub + branchName + "/" + remoteTrackBranch
```

Comment:
We can add a new method (`PublishedRef()`) for `ReviewableBranch`, and add testcases for it.